File: 백준/Bronze/25640. MBTI/MBTI.java
import java.io.BufferedReader;
import java.io.IOException;
import java.io.InputStreamReader;
public class Main {
    public static void main(String[] args)throws IOException{
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
        String mbti = br.readLine();
        int num = Integer.parseInt(br.readLine());
        int result = 0;
        for(int i = 0; i < num; i++){
            String str = br.readLine();
            if(str.equals(mbti)){
                result++;
            }
        }
        System.out.println(result);

    }
}

File: 백준/Bronze/25640. MBTI/MBTI.cs
string mbti = Console.ReadLine();
int num = int.Parse(Console.ReadLine());
int result = 0;
for (int i = 0; i < num; i++)
{
    string str = Console.ReadLine();
    if(str == mbti)
    {
        result++;
    }
}
Console.WriteLine(result);
